images/base/files/usr/local/bin/entrypoint

```diff
@@ -204,6 +204,10 @@ fix_cgroup() {
   # See: https://man7.org/linux/man-pages/man7/cgroups.7.html
   local current_cgroup
   current_cgroup=$(grep -E '^[^:]*:([^:]*,)?cpu(,[^,:]*)?:.*' /proc/self/cgroup | cut -d: -f3)
+  # on WSL2 systems /sys/fs/cgroup/systemd dir is missing which eventually leads to kubelet failing to start.
+  # see: https://github.com/kubernetes-sigs/kind/issues/2323
+  # the following mkdir command is a common workaround, see: https://github.com/microsoft/WSL/issues/4189#issuecomment-758439957
+  mkdir -p /sys/fs/cgroup/systemd/
```

Comment:
Technically I don't think this (cgroup controller heirarchy) has to be mounted at `/sys/fs/cgroup` though I've not ever encountered it mounted elsewhere.

see: https://man7.org/linux/man-pages/man7/cgroups.7.html

This directory would also normally be configured like `mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd`

Reply:
We actually already depend on it being under `/sys/fs/cgroup` below though, regarding the first point.

Reply:
... And it's not just kind assuming this in the ecosystem https://kubernetes.slack.com/archives/C0BP8PW9G/p1627412176214900

Reply:
awesome, that's good to know.

Reply:
as for the mounting, I believe this will be done by [this line](https://github.com/kubernetes-sigs/kind/pull/2390/files/12a8deeb38f5f11074cdc558b2a7f809a5242c67#diff-3c55751d83af635109cece495ee2ff38206764a8b95f4cb8f11fc08a5c0ea8dcR171) 
The current placement of this mkdir command is deliberate so that it gets picked up by the `cgroup_subsystems` on the next line and then passed to the `mount_kubelet_cgroup_root "/kubelet" "${subsystem}"`

At least, this is how it works for me. I've added just this one line and this allowed for the cluster to start up and results in the following mount being created by the entrypoint 
```
root@k8s-guide-control-plane:/# mount | grep kubelet | head -n 1
cgroup on /sys/fs/cgroup/cpuset/kubelet type cgroup (rw,nosuid,nodev,noexec,relatime,cpuset)
```